maestro-ios-xctest-runner/maestro-driver-iosUITests/Routes/Handlers/ViewHierarchyHandler.swift

```diff
@@ -118,11 +122,17 @@ struct ViewHierarchyHandler: HTTPHandler {
                 }
 
                 let alerts = logger.measure(message: "Fetch alert hierarchy") {
-                    fullScreenAlertHierarchy(element)
+                    fullScreenAlertHierarchy(element) {
```

Comment:
Is this call taking a function argument? I don't think this will compile can you try executing:

```
sh maestro-ios-xctest-runner/build-maestro-ios-runner.sh
```
Let's actually add a log on line 40 and 42 before we return because in case the endpoint returns error we would have the logs.